molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/impl/ResourcesImpl.java

```diff
@@ -61,4 +62,10 @@ public boolean hasRepository(String entityName)
 		}
 		return dataService.findAll(name, q);
 	}
+
+	public Set<String> getResourcesNames()
```

Comment:
It was needed to figure out of the resource is empty or the files were wrongly configured.
The error message thrown is the same in both conditions.